### ISSUE TITLE
Unable to build with Idris v0.12.3

### ISSUE THREAD

**Issue by pierrebeaucamp** [uwap/idris-http#11]
Hi,

I'm unable to build / install this library with Idris v0.12.3:

```
Can't find import Classes/Verified
```

`Class.Verified` was renamed to `Interfaces.Verified` in https://github.com/idris-lang/Idris-dev/commit/c3f0755519a651fd8bee9b1fc31755c917568116 but just renaming the import didn't do the job for me.

In https://github.com/idris-lang/Idris-dev/commit/f754d9c1b6de0cc9cb0aaa7f5f97f487a03d7087, `VerifiedMonad` changed from 

```
monadRightIdentity : (mx : m a) -> mx >>= Monad.return = mx
```

to 

```
monadRightIdentity : (mx : m a) -> mx >>= pure = mx
```

resulting in the following error:

```
./Http/RawResponse.idr:58:22:
When checking right hand side of Interfaces.Verified.Http.RawResponse.RawResponse implementation of Interfaces.Verified.VerifiedMonad, method monadRightIdentity with expected type
        MkRawResponse _ >>= pure = MkRawResponse _

Type mismatch between
        MkRawResponse _ = MkRawResponse _ (Type of Refl)
and
        pure _ = MkRawResponse _ (Expected type)

Specifically:
        Type mismatch between
                MkRawResponse _
        and
                pure _
```

I would have loved to resolve this myself and open a PR, but unfortunately, I was not able to detect the error in question. (I'm still quite new to the language)

Thank you for looking into this.

**Comment by uwap** [comment]
Thanks for reporting. Fixed with 7b0bbbc. :smile:

**Comment by pierrebeaucamp** [comment]
Thanks, that fixed it 🎉 

If you don't mind: could you point me towards the line in 7b0bbbc which fixed the `Type mismatch between MkRawResponse _ and pure _` error? It appears that you didn't have to change anything in `RawResponse.idr` itself. (I'm just curious)
